packages/core-data/src/fetch/__experimental-fetch-link-suggestions.ts

```diff
@@ -256,9 +256,12 @@ export default async function fetchLinkSuggestions(
  * a taxonomy title might be more relevant than a post title, but by default taxonomy results will
  * be ordered after all the (potentially irrelevant) post results.
  *
- * We sort by scoring each result, where the score is the number of tokens in the title that are
- * also in the search query, divided by the total number of tokens in the title. This gives us a
- * score between 0 and 1, where 1 is a perfect match.
+ * We sort by scoring each result, where the score is a combination of the number of exact matches
+ * and sub-matches for tokens in the title that are also in the search query, divided by the total
+ * number of tokens in the title. This gives us a score between 0 and 1, where 1 is a perfect match.
```

Comment:
> This gives us a score between 0 and 1, where 1 is a perfect match.

I don't fully understand the scoring logic yet, but the score can be greater than 1, right? Is the actual maximum score 10?

Reply:
Overall i can now be greater than 1. We could tweak the whole thing to keep scores below 1 if that helps?